packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -485,11 +485,17 @@ export class AtomicSearchInterface
   }
 
   private updateHash() {
-    history.pushState(
-      null,
-      document.title,
-      `#${this.urlManager.state.fragment}`
-    );
+    const newFragment = this.urlManager.state.fragment;
+
+    if (!this.searchStatus.state.firstSearchExecuted) {
```

Comment:
Should this technique be documented here ? https://docs.coveo.com/en/headless/latest/usage/synchronize-search-parameters-with-the-url/

Reply:
Yes this could be helpful, I'll open a jira for this